test/suites/consolidation/suite_test.go

```diff
@@ -239,30 +239,7 @@ var _ = Describe("Consolidation", func() {
 			// Update the deployment to only contain 1 replica.
 			env.ExpectUpdated(dep)
 
-			// Ensure that we get two nodes tainted, and they have overlap during consolidation
-			env.EventuallyExpectTaintedNodeCount("==", 2)
-			nodes = env.ConsistentlyExpectDisruptionsWithNodeCount(2, 5, 5*time.Second)
-
-			// Remove the finalizer from each node so that we can terminate
-			for _, node := range nodes {
-				Expect(env.ExpectTestingFinalizerRemoved(node)).To(Succeed())
-			}
-
-			// After the deletion timestamp is set and all pods are drained
-			// the node should be gone
-			env.EventuallyExpectNotFound(nodes[0], nodes[1])
-
-			// This check ensures that we are consolidating nodes at the same time
-			env.EventuallyExpectTaintedNodeCount("==", 2)
-			nodes = env.ConsistentlyExpectDisruptionsWithNodeCount(2, 3, 5*time.Second)
-
-			for _, node := range nodes {
-				Expect(env.ExpectTestingFinalizerRemoved(node)).To(Succeed())
-			}
-			env.EventuallyExpectNotFound(nodes[0], nodes[1])
-
-			// Expect there to only be one node remaining for the last replica
-			env.ExpectNodeCount("==", 1)
+			env.ConsistentlyExpectDisruptionsUntilTarget(2, 5, 0, 10*time.Minute)
```

Comment:
Is a struct a better representation for the input data? I'm confused what 2,5, and 0 mean here

Reply:
I added it since I figured it might help for readability

Reply:
Reverted, since it's just two ints now without a target int